packages/test/test-end-to-end-tests/src/test/gc/gcStateResetInSummaries.spec.ts

```diff
@@ -306,87 +306,6 @@ describeCompat("GC state reset in summaries", "NoCompat", (getTestObjectProvider
 		);
 	});
 
-	/**
-	 * This test validates that the GC state is regenerated if needed when state is refreshed from a snapshot. If GC is
-	 * enabled and state is refreshed from a snapshot that had GC disabled, the GC state needs to be reset and a full
-	 * tree summary should happen.
-	 */
-	it("regenerates GC state on refreshing from snapshot that has GC disabled", async () => {
```

Comment:
Removed this test because it does not test anything unique. This scenario is already tested in `gcLoadingFromOlderSummaries.spec.ts`

Reply:
I like these changes